requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+pandas
+numpy
+matplotlib
+tqdm
+requests
+xlrd
+tabulate
+yfinance
```

Comment:
we can add `-r requirements.txt`

Reply:
But then why do you want them separate?

Reply:
* `requirements-dev.txt` for developers this means if I'm a developer and I do `pip install -r requirementes-dev.txt` I will have my environment ready to testing, runing and developing code on this project.
* Sometimes, if we use a CI/CD environment, we only want install testing tools that's why `requirements-test.txt` exists. 
* If we can check a common user installation we can use `requirement-base.txt`.

Reply:
I see. 

What's the difference then between `requirements.txt` and `requirements-base.txt`?

Reply:
`requirements.txt` should be the concrete version (each package with their versions) of `requirements-base.txt` and ideally the `requiments.txt` defines the whole environment (100% reproducible) , this include all packages installed after checking that the library works properly (or all test had been passed).